modules/account-lib/test/unit/coin/dot/transactionBuilder/transferBuilder.ts

```diff
@@ -114,7 +114,7 @@ describe('Dot Transfer Builder', () => {
       should.deepEqual(txJson.tip, 0);
       should.deepEqual(txJson.transactionVersion, txVersion);
       should.deepEqual(txJson.chainName, chainName);
-      should.deepEqual(txJson.eraPeriod, 4);
+      should.deepEqual(txJson.eraPeriod, 0);
```

Comment:
When maxDuration value of `0` is passed when building a transaction, we know we want an immortal transaction. polkadot `17.15.1` added a new flag `isImmortalEra` to fix previous version where immortal transactions were being build incorrectly. When that flag is passed the `eraPeriod` is `0` which is the correct behavior now.